Development/pcm_block_alt.yml

```diff
@@ -0,0 +1,72 @@
+name: ConsIndShock
+
+states:
+  - &m
+    name: m
+    long_name: market resources
+    domain: [0, .inf]
+  - &a
+    name: a
+    long_name: assets
+    domain: [0, .inf]
+
+controls:
+  - &c
+    name: c
+    long_name: consumption
+    domain: [0, .inf]
+  - &stigma
+    name: stigma
+    long_name: risky portfolio share
+    domain: [0, 1]
+
+blocks:
+  - name: Consumption
+    states: [*m]
+    controls: [*c]
+    parameters: [beta, rho]
+    equations:
+      transitions:
+        - a: m - c
+        - v_0(m): u(c) + beta * v_1(a)
+    definitions:
+      - u(c): c^(1-rho) / (1-rho)
+
+  - name: Portfolio Choice
+    states: [*a]
+    controls: [*stigma]
+    equations:
+      transitions:
+        - v_1(a): v_2(a, stigma)
+
+  - name: Post-Decision
+    states: [*a, *stigma]

```

Comment:
interesting idea about how to define a variable, and then use it for both controls and states in different blocks.